comid-triples.cddl

```diff
@@ -0,0 +1,142 @@
+concise-mid-tag = {
+  ; TODO(tho) language
+  ; TODO(tho) tag-id
+  ; TODO(tho) module-name
+  ; TODO(tho) entity
+  ; TODO(tho) linked-tags
+  comid.claims => claims-map
+  ; TODO(tho) extension
+}
+
+claims-map = non-empty<{
+  comid.reference-claims => one-or-more<reference-claim-record>
+  comid.endorsed-claims => one-or-more<endorsed-claim-record>
+  comid.identity-claims => one-or-more<identity-claim-record>
+  * $$claims-map-extension
+}>
+
+; REFERENCE-VALUE triple
+; "target environment $TE" "has reference measurements" "$RV"
+reference-claim-record = {
+  target-environment: environment-map 
+  reference-values: values-map
+}
+
+; ENDORSED-VALUE triple
+; "environment $E" "has endorsed measurements" "$EV"
+endorsed-claim-record = {
+  environment: environment-map 
+  endorsed-values: values-map
```

Comment:
I am little nervous of putting a single map as "values-map" as a generic bag. I would prefer we keep Reference-value-map and Endorsed-value-map.

It may have identical entries to start with, however for clear separation of Reference Values against Endorsed Values (as per the RATs Architecture) I would prefer to keep them seperate!

This way any future addition to Reference Value Map does not pollute the Endorsed Value Map and Vice Versa!

Reply:
The value-map always exists in the context of en embedding endorsed or refval claim.  I don't expect value-map's to be shipped around in isolation (in which case your concern is a valid one), but always with the surrounding context.

Reply:
```suggestion
  endorsed-values: one-or-more<values-map>
```